### PR TITLE
Change "Accepted" to dropdown instead of checkbox

### PR DESCRIPTION
**Pull request by tylermercer**

### CODE REVIEW

**Comment by tylermercer** [comment]
I'm not convinced this is the best way to do this--specifically with having to explicitly convert `input.accepted` from a String to a bool in AddOfferForm's `commit` function. But when `accepted` was specified as a bool in the Yup schema, I couldn't get the dropdown to not be pre-filled.

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "react-relay": "^7.1.0",
     "react-relay-network-modern": "^4.3.0",
     "react-router-dom": "^5.1.2",
-    "react-scripts": "3.2.0",
+    "react-scripts": "^3.3.0",
```
Comment (bradenwatkins) [inline package.json]: Curious why this change?

Comment (tylermercer) [reply package.json]: Oh this was because I did `npm audit fix` after `npm i` warned me about vulnerabilities. Should I rebase to remove it from this PR?

**Comment by tylermercer** [comment]
The force-push was a rebase to remove the upgraded dependencies (since that's handled by Braden's issue that's already been merged)

File: package.json (hunk 1 above)
Comment (bradenwatkins) [reply package.json]: I think mine got merged so as long as there are no conflicts I think you're good to keep it.
